lib/testing/src/lib/core/pages/data-table-component.page.ts

```diff
@@ -103,6 +103,29 @@ export class DataTableComponentPage {
         await browser.actions().sendKeys(protractor.Key.NULL).perform();
     }
 
+    async selectMultipleRows(columnName: string, items: string[], columnPrefix = ''): Promise<void> {
+        await BrowserActions.pressEscape();
+        await this.clearRowsSelection();
+        await BrowserActions.pressCmd();
+        for (const item of items) {
+            await this.checkContentIsDisplayed(columnName, item, columnPrefix);
+            await this.selectRow(columnName, item, columnPrefix);
+        }
+        await BrowserActions.releaseKeyPressed();
+    }
+
+    async clearRowsSelection(): Promise<void> {
+        try {
+            const count = await this.getNumberOfSelectedRows();
+            if (count !== 0) {
+                await browser.refresh();
```

Comment:
refresh is quite problematic can we find another way?

Reply:
I could not find a different approach, but if there is any, I'm happy to apply it.

Reply:
why do you need it?

Reply:
If there is any row selected, it will be unselected, as part of selectMultipleRows() method, before selecting a set of items. 
We don't know what is selected, this is why we call a page refresh, to have a clean state, before performing the desired selection.